File: java64/nivel5/exer47.java
import javax.swing.JOptionPane;

public class exer47 {
    public static void main(String[] args) {
        int x, maior = 0, menor = 0, igual = 0;
        int[] numeros = new int[10];

        x = Integer.parseInt(JOptionPane.showInputDialog("Informe o valor de X:"));

        for (int i = 0; i < numeros.length; i++) {
            numeros[i] = Integer.parseInt(JOptionPane.showInputDialog("Informe  o " + (i + 1) + "° valor:"));
        }

        for (int y = 0; y < numeros.length; y++) {
            if (numeros[y] > x) maior++;
            if (numeros[y] < x) menor++;
            if (numeros[y] == x) igual++;
        }
        JOptionPane.showMessageDialog(null, "Maior quue X: " + maior + "\nMenor que X: " + menor + "\nIgual a X: " + igual);
    }
}

File: pyhton64/nivel5/exer47.py
vetor = []
x = int(input('Valor de X: '))
maior = menor = igual = 0

for i in range(1, 10 + 1):
    vetor.append(int(input('Informe valor: ')))

for z in range(len(vetor)):
    if vetor[z] > x:
        maior += 1
    elif vetor[z] < x:
        menor += 1
    elif vetor[z] == x:
        igual += 1

print('Vetor organizado:')
vetor.sort()
print(vetor)

print(f'\nMaiores que x: {maior}\nMenores que x: {menor}\nIguais a x: {igual}')
